web/portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -205,6 +196,11 @@
   </filter-mapping>
 
   <filter-mapping>
+    <filter-name>LocalizationFilter</filter-name>
+    <url-pattern>/*</url-pattern>
```

Comment:
Calling the filter and checking if an attribute exists in the Conversation will have performance impacts. The question is to know if it is a limited and acceptable impact or not.